packages/storybook-webpack-angular-types-plugin/src/lib/webpack-angular-types-plugin/type-extraction/ast-utils.ts

```diff
@@ -270,8 +267,24 @@ export function getDefaultValue(
 	if (decl instanceof PropertyDeclaration) {
 		const initializer = decl.getInitializer();
 		const initializerType = initializer?.getType();
-		if (initializer && initializerType && isEvaluableType(initializerType)) {
-			return initializer.getText();
+		if (initializer && initializerType) {
+			if (isEvaluableType(initializerType)) {
+				return initializer.getText();
+			}
+
+			// If the property is a InputSignal or ModelSignal and the initializer is a CallExpression,
+			// we extract the initializer's first argument, e.g. "42" from "input.required(42, options)".
+			if (
+				Node.isCallExpression(initializer) &&
+				(isInputSignal(decl) || isModelSignal(decl))
```

Comment:
Did you know there is a `isSignal` function from the `@angular/core` package? Can you use this instead?

Reply:
Do you think that this function works with the Typescript AST types (such as `CallExpression`)? I don't think so, but I can try that.

Reply:
Nope, doesn't work